eggplant/membership/tests.py

```diff
@@ -162,7 +158,7 @@ def test_send_invitation(self):
         expected = 'Invitation has been send to {}'.format(invited_email)
         self.assertContains(response, expected, 1, 200)
 
-        self.assertEqual(len(mail.outbox), 1)
+        self.assertEqual(len(mail.outbox), 1)  # @UndefinedVariable
```

Comment:
The helpful part is that I keep going back to this like because it has an error... so by putting that innocent little comment there, it switches off the error. I actually added it manually :)